nexus/db-queries/src/db/datastore/rack.rs

```diff
@@ -321,6 +312,31 @@ impl DataStore {
                     .map_err(TxnError::CustomError)?;
                 info!(log, "Created recovery silo");
 
+                // Create certificates in that Silo.
+                let certificates = certificates
+                    .into_iter()
+                    .map(|c| {
+                        Certificate::new(
+                            db_silo.id(),
+                            Uuid::new_v4(),
+                            ServiceKind::Nexus,
+                            c
+                        )
+                    })
+                    .collect::<Result<Vec<_>, _>>()
+                    .map_err(RackInitError::Certificate)
+                    .map_err(TxnError::CustomError)?;
+                {
+                    use db::schema::certificate::dsl;
+                    diesel::insert_into(dsl::certificate)
+                        .values(certificates)
+                        .on_conflict(dsl::id)
+                        .do_nothing()
```

Comment:
Does this mean there would be no error?

Reply:
Yeah.  The uuid was just generated, though, so it would be rather surprising if this happened.  But I'll go ahead and remove that.